index.html

```diff
@@ -3,11 +3,36 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>html css and java script</title>
   <link rel="stylesheet" href="style.css">
 </head>
 <body>
+  <header>
+    <ul class="nav-links">
+      <li><a href="https://www.facebook.com/login/?next=https%3A%2F%2Fwww.facebook.com%2F" target="_blank">Facebook <img src="/imgs/Facebook-logo.png" width="20" height="20" alt="fejs"></a></li>
+      <li><a href="https://www.instagram.com/" target="_blank">Instagram <img src="/imgs/2437730f7e3a5705e205e67fa2cd1020.jpg" width="20" height="20" alt="ig"></a></li>
+      <li><a href="https://x.com/?lang=en" target="_blank">X <img src="/imgs/x-twitter-logo-on-black-circle-5694247.png" width="20px" height="20px" alt="xlogo"></a></li>
+    </ul>
+  </header>
   <h1>hello world</h1>
+  <div class="container">
+    <div class="klasa" id="klasa1"></div>
+    <div class="klasa" id="klasa2"></div>
+    <div class="klasa" id="klasa3"></div>
+    <p class="paragraf"></p>
+  </div>
+  <button class="buttoon">
+    <div class="box"></div>
+  </button>
+  <div class="pic-container">
+    <div class="pic1"><img src="/imgs/memes-real-madrid-barcelona-24-710x474.jpg" width="200" height="200" alt="sliba"></div>
+    <div class="pic2"><img src="/imgs/PNG_transparency_demonstration_1.png" width="200" height="200" alt="sliba2"></div>
+  </div>
   <script src="script.js"></script>
```

Comment:
Why `script`  tag is not at the end of `body` tag

https://stackoverflow.com/a/17140419